examples/timeSeries-stock-prediction/index.html

```diff
@@ -0,0 +1,22 @@
+<!--
+  👋 Hello! This is an ml5.js example made and shared with ❤️.
+  Learn more about the ml5.js project: https://ml5js.org/
+  ml5.js license and Code of Conduct: https://github.com/ml5js/ml5-next-gen/blob/main/LICENSE.md
+ 
+  This example demonstrates Training a Stock Price Predictor through ml5.TimeSeries.
```

Comment:
Hi professor, changed the example to weather predictor as you said, it works quite well and the dataset I put makes for dynamic predictions, however I am not sure how to visualize it, currently just making a table of values

Reply:
Have you considered plotting them along the x axis?

Reply:
I could plot it but it was the exact problem I had with the stock predictor where the code was getting complicated just because of the UI, that said, I implemented a simple dynamic table that shows only the latest values, so it still has a responsive feel to it. you can check it out in the example if you'd like!